packages/teams-js/src/private/videoEx.ts

```diff
@@ -104,7 +104,9 @@ export namespace videoEx {
             () => {
               notifyVideoFrameProcessed(timestamp);
             },
-            notifyError,
+            (errorMessage: string) => {
+              notifyError(`[VideoFrameProcessingFailed]: ${errorMessage}`);
+            },
```

Comment:
If the expected outcome of calling `notifyError` is to stop the video pipeline, it might be risky to use it here as it hadn't been that way from a dev's perspective.

Rather, we could expose `notifyError` as a standalone API, which accepts different levels of errors (e.g., Fatal, Alert, Minor, etc.) and the Teams client can decide on what to do based on the error level.

For frame processing failure, we may use `Alert` or `Minor` levels for backward compatibility